src/helper/calculateSlippage.ts

```diff
@@ -0,0 +1,7 @@
+export const calculateSlippage = (base: string, percent: number | undefined) => {
+  if (percent === undefined) {
+    throw new Error("Percent is undefined. Please provide a valid percent value.");
+  }
```

Comment:
Can u put this under index.ts? We do not want to have 1 function in one file only

Reply:
I do not think we need to throw a new Error here. Just return null